packages/adminrouter/extra/src/includes/server/master.conf

```diff
@@ -189,6 +189,9 @@ location ~ ^/service/(?<service_path>.+) {
     include includes/proxy-headers.conf;
     include includes/websockets.conf;
 
+    # Disable request buffering to support large files
+    # proxy_request_buffering off;
```

Comment:
This is commented. Is this on purpose?

Reply:
Ouch, good catch!

Reply:
Sorry, I missed that too. Fixed this and created https://github.com/dcos/dcos/pull/2408 (as I didn't like merging EE pr with no diff)

Reply:
> as I didn't like merging EE pr with no dif

@takirala  - I think you are mistaken with an understanding of EE PR here. Never an EE PR with no-diff (or upstream.json pointing your PR/fork/branch) is merged. The bump-ee tests the EE against your PR, and then our automation makes certain that when the Open is merged, your EE PR is updated with merged-SHA and merged immediately. It's like merging a pair of PRs, second one having a reference to the HEAD of the first).